examples/tabs/palette/src/components/HowToComponent.tsx

```diff
@@ -1,26 +1,24 @@
 import { H1 } from "@blueprintjs/core"
 import * as React from "react"
-export const HowToComponent = () => {
-  return (
```

Comment:
Why did this change?

Reply:
Inlines the return for a simpler code block since the entire return is JSX and there's no typescript orchestration code before it.